components/Banner/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -303,4 +303,40 @@ - (void)testDynamicTypeForContentSizeCategoryExtraExtraLarge {
   }
 }
 
+- (void)testDynamicTypeForAttributedTextStringWhenContentSizeCategoryIsExtraExtraLarge {
+  if (@available(iOS 10.0, *)) {
+    // Given
+    self.bannerView = [[MDCBannerView alloc] init];
+    self.typographyScheme =
+        [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201902];
+
+    // When
+    NSMutableAttributedString *bannerString =
+        [[NSMutableAttributedString alloc] initWithString:kBannerShortText];
+    [bannerString addAttribute:NSFontAttributeName
+                         value:self.typographyScheme.body1
+                         range:NSMakeRange(10, 8)];
+    [bannerString addAttribute:NSForegroundColorAttributeName
+                         value:UIColor.redColor
+                         range:NSMakeRange(0, 9)];
+    [bannerString addAttribute:NSLinkAttributeName
+                         value:@"http://www.google.com"
+                         range:NSMakeRange([kBannerShortText length] - 2, 2)];
+    self.bannerView.textLabel.font = self.typographyScheme.body2;
+    self.bannerView.textLabel.attributedText = bannerString;
+    MDCButton *button = self.bannerView.leadingButton;
```

Comment:
Please move the non-attributed string code into the "Given" section, since it appears the purpose of this test is to verify the layout when attributed text is provided.

Reply:
Addressed in d5953cafef899e0d094e3319e91bd446d0f7fc43.